src/aosm/azext_aosm/tests/latest/test_aosm_cnf_publish_and_delete.py

```diff
@@ -1,91 +1,118 @@
-# --------------------------------------------------------------------------------------------
+# # --------------------------------------------------------------------------------------------
 # Copyright (c) Microsoft Corporation. All rights reserved.
 # Licensed under the MIT License. See License.txt in the project root for license information.
+#
+# This is an integration tests for the aosm extension. It tests the following commands for the
+# cnf definition type:
+#   aosm nfd build
+#   aosm nfd publish
+#   aosm nfd delete
+#   aosm nsd build
+#   aosm nsd publish
+#   aosm nsd delete
+#
 # --------------------------------------------------------------------------------------------
 
-# Currently commented out because of the timeout bug in the testing framework.
+import os
+from typing import Dict
+from azure.cli.testsdk import LiveScenarioTest, ResourceGroupPreparer
+from knack.log import get_logger
+from jinja2 import Template
+from .recording_processors import TokenReplacer, SasUriReplacer
```

Comment:
I think you'll need to run isort on this list

Reply:
according to my linting they are in the correct order. The recording processors import was redundant so I removed it.